dev/io.openliberty.microprofile.health.3.0.internal_fat/test-applications/DelayedHealthCheckApp/src/io/openliberty/microprofile/health30/delayed/health/check/app/DelayedServlet.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.health30.delayed.health.check.app;
+
+import java.io.IOException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ *
+ */
+public class DelayedServlet extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public DelayedServlet() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+    @Override
+    public void init() {
+        System.out.println("Entering init function - Starting Thread.sleep for 30 seconds.");
+        try {
+            Thread.sleep(65000); //30 seconds = 30000 ms
```

Comment:
This actually sleeps for 65 seconds.  It doesn't match the println above and the comment on the right.